README.md

```diff
@@ -31,7 +31,8 @@ The following documentation shows you an overview of the workflow, and the resou
 
 ### Installing and interacting with the `ilab` CLI tool
 
-The `ilab` tool allows you to interact with the IBM AI model `Merlinite` or `Granite`, contribute your own information, and train the model locally.
+The `ilab` tool allows you to interact with the [Granite](https://en.wikipedia.org/wiki/IBM_Granite),
```

Comment:
Perhaps [https://github.com/IBM-Granite](https://github.com/IBM-Granite) or [https://huggingface.co/ibm-granite](https://huggingface.co/ibm-granite) would be better links than Wikipedia.